rtil/src/threads/ue/rebo/livesplit.rs

```diff
@@ -97,4 +110,34 @@ impl Run {
     pub fn attempt_count() -> u32 {
         LIVESPLIT_STATE.lock().unwrap().timer.run().clone().attempt_count()
     }
+    pub fn save_splits(path: String) -> Result<(), SplitsSaveError> {
+        let state = LIVESPLIT_STATE.lock().unwrap();
+        let mut timer = state.timer.clone();
+        let filename = Path::new(&path).file_name().unwrap().to_str().unwrap().to_owned();
```

Comment:
The `unwrap`s here indicate errors and should thus use `expect` instead for better debugging / better user communication in the logs.